dataset/dictionary/characters.json5

```
@@ -5203,6 +5210,22 @@
     zhCN: "瓦雷莎",
     tags: [ "natlan", "character-main" ],
   },
+  {
+    en: "Ifa",
+    ja: "イファ",
+    zhCN: "伊法",
+    notesEn: "Veterinarian from the \"Flower-Feather Clan\"",
+    notes: "「花翼の集」の獣医",
+    notesZh: "「花羽会」中的兽医",
+    tags: [ "natlan", "character-main" ],
+  },
+  {
+    en: "Cacucu",
+    ja: "カクーク",
+    zhCN: "咔库库",
+    notes: "イファが連れているクク竜",
+    tags: [ "natlan", "character-main" ],
```

The companion beasts (like Guoba) are currently categorized as `character-sub`. Let me change the tag for now.

```suggestion
    tags: [ "natlan", "character-sub" ],
```